client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -43,6 +46,67 @@ const CancelPurchaseRefundInformation = ( {
 		} );
 	};
 
+	const featuresList = ( productSlug, hasDomain ) => {
```

Comment:
Similar to one of the comments above, this function is effectively a component: it's a function which accepts arguments (props) and returns HTML.

Could this be moved into its own component? React will do a better job of optimising it, its dependencies will be more clear, and it will be more readable.

Reply:
`featuresList` has been move outsite the main component